mch-questions.yml

```diff
@@ -64,6 +75,9 @@ questions:
       [this Mojang Support article](https://help.mojang.com/customer/portal/articles/361483) for information on how to
       get your account back.
 
+      Note that our volunteers **have no access to account information**, and can't help you with account issues.
+      If you already sent a email to Mojang, please wait for their response.
```

Comment:
We should link the form to email Mojang as well here, to push it toward users more.

Reply:
Done, I made "sent a email to mojang" a link